chart/etcd-backup-restore/values.yaml

```diff
@@ -60,6 +60,7 @@ backup:
   garbageCollectionPeriod: "1m"
 
   etcdConnectionTimeout: "30s"
+  etcdSnapshotTimeout: "180s"
```

Comment:
I suggest to keep this even longer by default. We could technically have 8GB snapshots.

Reply:
agreed